UI.tsv

```diff
@@ -325,7 +325,7 @@ UI_20150317_000324	{@st56}Dead{/}
 UI_20150317_000325	Spell Power
 UI_20150317_000326	Defense
 UI_20150317_000327	Magic Resistance
-UI_20150317_000328	Critical Rate
+UI_20150317_000328	Critical Attack Chance
```

Comment:
I disagree. 'Critical Attack Chance' is wrong. It's a number value. I don't see a problem with 'Critical Rate'. 'Critical Attack XXX' is too long in my opinion. I believe this is already addressed in skill tooltips and currently it doesn't align perfectly. 

'Critical Rate' vs 'Critical Attack Rate'. I don't know.

Also, I am not sure if this concept lives on, but Tree of Savior will have a wiki that you can access within the game. There is already the help topics that can be accessed through a menu that tells about the functions of the game. Though, I have not encountered a 'Help Topic' concerning this.